pkg/analyzer/analyzers/launchdarkly/launchdarkly.go

```diff
@@ -0,0 +1,203 @@
+	if info == nil {
+		return nil
+	}
+
+	result := analyzers.AnalyzerResult{
+		AnalyzerType: analyzers.AnalyzerTypeElevenLabs,
+		Metadata:     map[string]any{},
+		Bindings:     make([]analyzers.Binding, 0),
+	}
+
+	// extract information from resource to create bindings and append to result bindings
+	for _, resource := range info.Resources {
+		binding := analyzers.Binding{
+			Resource: *secretInfoResourceToAnalyzerResource(resource),
+			Permission: analyzers.Permission{
+				Value: getPermissionType(info.User.Token),
+			},
+		}
+
+		if resource.ParentResource != nil {
+			binding.Resource.Parent = secretInfoResourceToAnalyzerResource(*resource.ParentResource)
+		}
+
+		result.Bindings = append(result.Bindings, binding)
+
+	}
+
+	return &result
+}
+
+// secretInfoResourceToAnalyzerResource translate secret info resource to analyzer resource for binding
+func secretInfoResourceToAnalyzerResource(resource Resource) *analyzers.Resource {
+	analyzerRes := analyzers.Resource{
+		FullyQualifiedName: resource.ID,
+		Name:               resource.Name,
+		Type:               resource.Type,
+		Metadata:           map[string]any{},
+	}
+
+	for key, value := range resource.MetaData {
+		analyzerRes.Metadata[key] = value
+	}
+
+	return &analyzerRes
+}
+
+// getPermissionType return what type of permission is assigned to token
+func getPermissionType(token Token) string {
+	permission := ""
+
+	if token.Role != "" {
+		permission = token.Role
+	} else if token.hasInlineRole() {
+		permission = "Inline Policy"
+	} else if token.hasCustomRoles() {
+		permission = "Custom Roles"
+	}
+
+	return permission
+}
+
+// printUser print User information from secret info to cli
+func printUser(user User) {
+	// print caller information
+	color.Green("\n[i] User Information:")
+	callerTable := table.NewWriter()
+	callerTable.SetOutputMirror(os.Stdout)
+	callerTable.AppendHeader(table.Row{"Account ID", "Member ID", "Name", "Email", "Role"})
+	callerTable.AppendRow(table.Row{color.GreenString(user.AccountID), color.GreenString(user.MemberID),
+		color.GreenString(user.Name), color.GreenString(user.Email), color.GreenString(user.Role)})
+
+	callerTable.Render()
+
+	// print token information
+	color.Green("\n[i] Token Information")
+	tokenTable := table.NewWriter()
+	tokenTable.SetOutputMirror(os.Stdout)
+
+	tokenTable.AppendHeader(table.Row{"ID", "Name", "Role", "Is Service Token", "Default API Version",
+		"No of Custom Roles Assigned", "Has Inline Policy"})
+
+	tokenTable.AppendRow(table.Row{color.GreenString(user.Token.ID), color.GreenString(user.Token.Name), color.GreenString(user.Token.Role),
+		color.GreenString(fmt.Sprintf("%t", user.Token.IsServiceToken)), color.GreenString(fmt.Sprintf("%d", user.Token.APIVersion)),
+		color.GreenString(fmt.Sprintf("%d", len(user.Token.CustomRoles))), color.GreenString(fmt.Sprintf("%t", user.Token.hasInlineRole()))})
+
+	tokenTable.Render()
+
+	// print custom roles information
+	if user.Token.hasCustomRoles() {
```

Comment:
nit: we can use an early return to reduce indentation
```suggestion
    if !user.Token.hasCustomRoles() {
        return
    }
    // print token information
    color.Green("\n[i] Custom Roles Assigned to Token")
    ...
```